client/jetpack-cloud/sections/partner-portal/searchable-dropdown/style.scss

```diff
@@ -0,0 +1,61 @@
+.searchable-dropdown {
+	.components-base-control__field.components-base-control__field {
+		margin: 0;
+	}
+
+	input.components-combobox-control__input[type="text"] {
+		font-size: 1rem;
+		line-height: 1.5;
+		padding: 7px 14px;
+	}
+
+	.components-combobox-control__suggestions-container .components-flex {
+		border: 1px solid var(--color-neutral-10);
+		border-radius: 2px;
+		background-color: var(--color-surface);
+		transition: all 0.15s ease-in-out;
+		height: 40px;
```

Comment:
It's a real bummer we have to override so many styles. 😞 

I wonder if some of them can be centrally applied to properly adapt and style the combobox control better for the entirety of Calypso 🤔

Reply:
I totally agree with this. I would say we can maybe do this on a separate PR. There are a couple of areas the combo box is being used which have similar styling. we can probably decide what would be the standard calypso styling from there.

Reply:
Sounds good, no need to address it here. Perhaps we can add a TODO comment then to remember to address it.

Reply:
This has been added. 👍